public/app/features/explore/Logs/Logs.tsx

```diff
@@ -300,7 +303,7 @@ const UnthemedLogs: React.FunctionComponent<Props> = (props: Props) => {
         }
       });
     }
-  }, [logsVolumeData?.data, unregisterAllChildren, logsVolumeEnabled, hiddenLogLevels, register, toggleLegendRef]);
+  }, [logsVolumeData, unregisterAllChildren, logsVolumeEnabled, hiddenLogLevels, register, toggleLegendRef]);
```

Comment:
Won't this change the render behavior of this component? If something mutates data of `logsVolumeData` this won't re-render anymore, only if the `logsVolumeData` reference is updated?

Reply:
That shouldn't happen and works as expected in my tests, but the signature of this PR already grew beyond what it should have been given it's for an escalation and the thrown exceptions, so let's keep the original dependency array.